# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2025/01
# disable FetchContent for pybind11, abseil-cpp, pybind11_abseil, cifpp and dssp
# use find_package instead.
diff -ru alphafold3-3.0.0/CMakeLists.txt alphafold3-3.0.0_disable_fetch/CMakeLists.txt
--- alphafold3-3.0.0/CMakeLists.txt	2024-11-11 11:42:47.000000000 +0100
+++ alphafold3-3.0.0_disable_fetch/CMakeLists.txt	2025-01-14 17:14:41.925312870 +0100
@@ -54,8 +54,11 @@
   GIT_TAG 57560472b4260dc41f457706bc45fc6ef0bc0f10 # v4.4.7
   EXCLUDE_FROM_ALL)
 
-FetchContent_MakeAvailable(pybind11 abseil-cpp pybind11_abseil cifpp dssp)
-
+#FetchContent_MakeAvailable(pybind11 abseil-cpp pybind11_abseil cifpp dssp)
+find_package(pybind11 CONFIG REQUIRED)
+find_package(absl CONFIG REQUIRED)
+find_package(pybind11_abseil CONFIG REQUIRED)
+find_package(dssp CONFIG REQUIRED)
 find_package(
   Python3
   COMPONENTS Interpreter Development NumPy
